<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>570</y>
      <width>721</width>
      <height>67</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="sendBtn">
       <property name="maximumSize">
        <size>
         <width>93</width>
         <height>31</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="autoBtn">
       <property name="maximumSize">
        <size>
         <width>93</width>
         <height>31</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>自动</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="pTxtEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>721</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>721</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="title">
     <string>请选择要传输的文件</string>
    </property>
    <widget class="QComboBox" name="filePathBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>551</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="selectBtn">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>34</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>文件路径：</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>721</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="title">
     <string>文件传输进度条</string>
    </property>
    <widget class="QProgressBar" name="sendProg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>691</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>501</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="title">
     <string>请输入目的网址和端口号</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="ipLabel">
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string>目的网址：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ipLineEdit">
       <property name="maximumSize">
        <size>
         <width>241</width>
         <height>31</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="portLabel">
       <property name="maximumSize">
        <size>
         <width>68</width>
         <height>29</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>端口号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="portLineEdit">
       <property name="maximumSize">
        <size>
         <width>61</width>
         <height>31</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>721</width>
      <height>78</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="title">
     <string>请选择要传输文件所在的文件夹</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>文 件 夹:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="folderComboBox">
       <property name="minimumSize">
        <size>
         <width>551</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>551</width>
         <height>31</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="selectFolderBtn">
       <property name="minimumSize">
        <size>
         <width>31</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
